File: C#/백준/Bronze/6764. Sounds fishy！/Sounds fishy！.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] a = new int[4];
            for (int i=0; i<4; i++){
                a[i] = int.Parse(Console.ReadLine());
            }
            if(a.Max()==a.Min()){
                Console.WriteLine("Fish At Constant Depth");
            }
            else if(a[0]<a[1] && a[1]<a[2] && a[2]<a[3]){
                Console.WriteLine("Fish Rising");
            }
            else if(a[0]>a[1] && a[1]>a[2] && a[2]>a[3]){
                Console.WriteLine("Fish Diving");
            }
            else{
                Console.WriteLine("No Fish");
            }
        }
    }
}

File: Python/백준/Bronze/6764. Sounds fishy！/Sounds fishy！.py
aa=[]
for i in range(4):
    aa.append(int(input()))
if(max(aa)==min(aa)):
    print('Fish At Constant Depth')
elif(aa[0]<aa[1] and aa[1]<aa[2] and aa[2]<aa[3]):
    print('Fish Rising')
elif(aa[0]>aa[1] and aa[1]>aa[2] and aa[2]>aa[3]):
    print('Fish Diving')
else:
    print('No Fish')